### DISCUSSION TITLE
boxShadow: ['dark'] variant

### DISCUSSION THREAD

**Discussion by michahell** [tailwindlabs/tailwindcss#5169] [Ideas]
Hello,

I use the `dark` (class) modifier to change styling based on [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
I use `shadow-md` both in `light` and `dark` modes, however I would like to modify the way shadow looks in dark mode: either remove it, or make it a lot heavier, for example.
Ideally, what I'd like to use, is `dark:shadow-md` for instance.
I tried adding it to my `tailwind.config.js` as follows:

```
variants: {
    extend: {
      boxShadow: ['dark']
    },
  },
```

But this does not work.
Am I missing something, or would this be a feature request?

**Comment by alexanderbluhm** [comment]
Hi @michahell 
I am not 100% sure what you mean because you say
> I use `shadow-md` both in light and dark modes. 

So, if you use the same class on both modes, you won't see any difference.
Instead, you can do something like this to make a different shadow in dark mode:
https://play.tailwindcss.com/plrnnI3Tvj
